fr.md

```diff
@@ -367,8 +367,7 @@ Ces sites donnent de nombreuses informations fausses et/ou obsolètes et ne devr
 - OpenClassrooms
 - W3Schools
 - W3Resource
-- La chaîne youtube de PrimFX
-- La chaîne youtube de Graven - Développement
```

Comment:
Depreciate contents doesn't mean any hain against the author. This PR seems to be reasonless.